tests/booking-form.html

```diff
@@ -105,7 +105,14 @@ <h2>Booking Form 3</h2>
                     ]'>
             ]'>
         </irnmn-location>
-        
+
+        <irnmn-calendar id="booking-calendar2" label="Booking Dates" name="booking_another" placeholder="Choose your date" format-date-values="YYYY-MM-DD"
```

Comment:
@simide-ahnl I think everything we need to pass to a component will need to be inside translate strings.

Reply:
yes, but this file is just for testing